lib/ansible/modules/monitoring/grafana_plugin.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017, Thierry Sallé (@tsalle)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'metadata_version': '1.1'
+}
+
+DOCUMENTATION = '''
+---
+module: grafana_plugin
+author:
+  - "Thierry Sallé (@tsalle)"
+version_added: "2.5"
+short_description: Manage grafana plugins via grafana-cli
+description:
+  - install / remove grafana plugins
+options:
+  name:
+    required: true
+    description:
+      - Name of the plugin
+  version:
+    description:
+      - Version of the plugin to install.
+      - Default to latest
+  grafana_plugins_dir:
+    description:
+      - Directory where grafana plugin will be installed.
+  grafana_repo:
+    description:
+      - Grafana repository ("https://grafana.net/api/plugins")
```

Comment:
Should `https://grafana.net/api/plugins` be the `default:` (and in argspec)?`

Reply:
No, this option is only used when we want to use a custom grafana repository. (cf. https://github.com/grafana/grafana/pull/9299).
I've updated the description..